bazel/external_crates.bzl

```diff
@@ -630,7 +630,7 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
             ),
             "ic-http-gateway": crate.spec(
                 git = "https://github.com/dfinity/http-gateway",
-                rev = "3be26b5a2c71bf56e05b910951c1935a1ac550c4",
+                tag = "0.1.0-b0",
```

Comment:
It doesn't really matter, everyone can stay on the hash. Tag is just more informative, at least to me. And if @nathanosdev will keep them in semver manner then it'll be easier. Or, better, publish at crates.io

Reply:
Tags are much simpler for consumers than commit shas because that's an easy way that we can advertise the "safe" git shas to use. We will be making code changes in between this git tag and the next one so there will be a number of "unsafe" git shas in between the two git tags.

We will follow semver with these tags until we publish the crate on crates.io, and then continue the same versioning scheme on crates.io.

Reply:
@cgundy Does the above motivation make sense to you?

Reply:
Sounds good! I've just updated the PR description.